lib/utils/mismatchUtils.js

```diff
@@ -20,26 +20,35 @@ const { XMLXSDValidator } = require('../xsdValidation/XMLXSDValidator'),
  * @returns {object} an instance of validator Library
  */
 function useValidator() {
-  const lib = new XMLXSDValidator('xmllint').getValidatorLibrary();
+  const lib = new XMLXSDValidator('libxmljs').getValidatorLibrary();
```

Comment:
actually this is the one that fails once packed web version use this instead libxmljs2

Reply:
Done, thanks!!